extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1285,12 +1286,21 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
         'date=${timestamp}&' +
         'ampid=${clientId(_parsely_visitor)}',
       'pageview': '${basePrefix}&action=pageview',
+      'heartbeat': '${basePrefix}&action=heartbeat' +
+      '&tt=${totalEngagedTime}&inc=${incrementalEngagedTime}',
     },
     'triggers': {
       'defaultPageview': {
         'on': 'visible',
         'request': 'pageview',
       },
+      'heartbeat': {
+        'on': 'timer',
+        'timerSpec': {
+          'interval': 5,
```

Comment:
in this use case, is the `incrementalEngagedTime` always 5s?
I'm asking because we're promising a `$timerDuration` var to the Timer trigger, which shows the time duration since last timer trigger. Once that is landed, are you able to just use that?

Reply:
See the proposal about timerDuration here: https://github.com/ampproject/amphtml/issues/11290

Reply:
In our non-AMP implementation we send a heartbeat to the server every N seconds, that `interval` represents the N in this implementation. That doesn't necessarily mean that the user was engaged for 5 seconds. They may have been engaged for 2 seconds but we'll fire the heartbeat request every 5 seconds and send back that incrementalEngagedTime value. Does that make sense?

Our non-AMP implementation does support a configurable heartbeat delay so if `$timerDuration` would give us that then AMP engagement tracking would be more analogous which would be excellent.

Reply:
> In our non-AMP implementation we send a heartbeat to the server every N seconds, that interval represents the N in this implementation. That doesn't necessarily mean that the user was engaged for 5 seconds. They may have been engaged for 2 seconds but we'll fire the heartbeat request every 5 seconds and send back that incrementalEngagedTime value. Does that make sense?

That makes sense.

> Our non-AMP implementation does support a configurable heartbeat delay so if $timerDuration would give us that then AMP engagement tracking would be more analogous which would be excellent.

what kind of configuration? can you give an example?

Reply:
Sorry, I forgot that all the URL substitution variables come from the code and aren't configurable by the client. 

In our non-AMP implementation of engaged time tracking our customers can specify a number of milliseconds between heartbeats. I was thinking that we could use the `$timerDuration` to implement that, but I think I was just confusing myself.